nv

```diff
@@ -0,0 +1,19 @@
+#!/bin/sh -e
+
+# Helper script to start nvim with offset configuration by placing a
+# copied script in your ~/.local/bin/.
+
+# If you installed NvChad to ~/.config/nvim_NvChad, you can use this
+# script as is.
+
+# Otherwise, please edit to match your situation.
+
+HOME=~
+PROJECT_BIN="nvim"           # binary executable name for nvim
+PROJECT_NAME="nvim-NvChad"   # configuration data location (parent only)
+
+# If you put your configuration data elsewhere other than the normal
+# ~/.config, rewrite this to set it with the full path.
+GIT_CLONE_DIR=${XDG_CONFIG_HOME:-$HOME/.config/$PROJECT_NAME}
+
+exec "${PROJECT_BIN}" -u "${GIT_CLONE_DIR}/init.lua" "$@"
```

Comment:
Hmmmm,

not sure how I feel about making a shell executable to be moved outside of the `~/.config/nvim/` or `~/.config/nvim-NVCHAD/` folder 🤔 

I guess it feels a little outside the scope in a sense?

Reply:
I understand your initial negative feeling.  As long as nvim is not started with -u VIMRC, it reads normal files.

Having safe alternative configuration always available and coexisting is a nice feature though.

Reply:
Yeah, I can totally agree that there are some cool benefits to this!

And also we all greatly appreciate your 3 PRs!!!

I think the main thing is we want to ensure we have a good balance of complexity vs. simplicity in NvChad.

Let's keep discussing & thinking about this, it's v. cool